app/controllers/integrations/base_controller.rb

```diff
@@ -132,7 +132,7 @@ def create_docker_images
   end
 
   def record_log(level, message)
-    (@recorded_log ||= "".dup) << "#{level.upcase}: #{message}\n"
+    (@recorded_log ||= +"") << "#{level.upcase}: #{message}\n"
```

Comment:
This is weird. Is this string concatenation or some other thing?

Reply:
rubocop rule is to prefer `+""` over `"".dup`

Reply:
Yeah, seems faster too. http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Performance/UnfreezeString

Okay. I buy it.